frontend/src/metabase/components/MetadataInfo/Popover/Popover.styled.tsx

```diff
@@ -4,17 +4,9 @@ import { HoverCard } from "metabase/ui";
 
 export const WidthBound = styled.div<{ width?: number }>`
   font-size: 14px;
-  width: ${props => props.width ?? 300}px;
+  max-width: ${props => props.width ?? 300}px;
 `;
 
 export const Dropdown = styled(HoverCard.Dropdown)`
   overflow: visible;
 `;
-
-export const HackyInvisibleTargetFiller = styled.div`
```

Comment:
This hack only worked if the popover was displayed below the trigger, which is not the case in filter modal.

Screenshot shows this element with `background: red;` added:

![image](https://github.com/user-attachments/assets/34ac4a24-d1a8-40da-9b44-ee743d7dd2e2)